src/lib/menu/menu.spec.ts

```diff
@@ -311,6 +371,22 @@ class PositionedMenu {
   @ViewChild('triggerEl') triggerEl: ElementRef;
 }
 
+interface TestableMenu {
+  trigger: MdMenuTrigger;
+  triggerEl: ElementRef;
+}
+@Component({
+  template: `
+    <button [mdMenuTriggerFor]="menu" #triggerEl>Toggle menu</button>
+    <md-menu overlap-trigger="false" #menu="mdMenu">
+      <button md-menu-item> Not overlapped Content </button>
+    </md-menu>
+  `
+})
+class NonOverlapMenu implements TestableMenu {
+  @ViewChild(MdMenuTrigger) trigger: MdMenuTrigger;
```

Comment:
It's being used to open the menu: https://github.com/trshafer/material2/blob/5373c66e8cd9d0a06485158434d0afbc5233245a/src/lib/menu/menu.spec.ts#L271